doc/lspconfig.txt

```diff
@@ -185,28 +195,31 @@ The most commonly passed overrides to `setup {}` are:
 
 - {on_attach} `function(client, bufnr)`
 
-  Callback invoked by neovim's built-in client when attaching a buffer to a
-  language server. Allows setting options (local or otherwise) conditioned
-  after a language server attaches, or overriding client options. Most
-  commonly used for settings buffer local keybindings.
+  Callback invoked by Nvim's built-in client when attaching a buffer to a
+  language server. Often used to set Nvim (buffer or global) options or to
+  override the Nvim client properties (`resolved_capabilities`) after a
+  language server attaches. Most commonly used for settings buffer
+  local keybindings. See |lspconfig-keybindings| for a usage example.
 
 - {settings} `table <string, string|table|bool>`  
 
   The `settings` table is sent in `on_init` via a
-  `workspace/didChangeConfiguration` notification from the neovim client to
+  `workspace/didChangeConfiguration` notification from the Nvim client to
   the language server. These settings allow a user to change optional runtime
   settings of the language server. 
 
-  The lspconfig document generation pipeline converts the `package.json`
-  distributed by many language servers that includes optional settings into
-  documentation in `server_configurations.md`. As a worked example, to send
-  the following:
+  The scripts that automatically generate `server_configurations.md` convert 
+  the `package.json` referened in many a server's configuration source file
```

Comment:
"referenced"

Reply:
How many "scripts" are there? isn't it just the one? Also, wouldn't it be better to make it clear that this is more or less specific to servers implemented in typescript/javascript?

Reply:
it's not tho